packages/material-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rjsf/material-ui",
-  "version": "3.2.1",
+  "version": "3.2.2",
```

Comment:
Can you undo these version bump changes

Reply:
Hi @epicfaace 
I updated the code to undo these version changes. Thank you,